modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/rest-openapi.yaml

```diff
@@ -5401,6 +5401,190 @@ paths:
                       - HeadlessCommerceAdminCatalog.write
             summary: Creates or updates a value related to a product option.
             tags: ["ProductOptionValue"]
+    /productSpecifications/{id}:
+        delete:
+            operationId: deleteProductSpecification
+            parameters:
+                - in: path
+                  explode: false
+                  name: id
+                  required: true
+                  schema:
+                      description:
+                          The resource Id
+                      example: 30130
+                      format: int64
+                      minimum: 0
+                      type: integer
+                  style: simple
+            responses:
+                204:
```

Comment:
```suggestion
                204:
                    content:
                        application/json: {}
                        application/xml: {}
```

Reply:
In this case I don't get the point... the 204 http code is "No content", you should not return an empty object (otherwise you have content).
Also the other delete API have the same structure

Example: 
        delete:
            operationId: deleteSkuByExternalReferenceCode
             ....
              responses:
                204:
                    description:
                        No content
                401:

Reply:
Changed as suggested. I did it "in the old way"

Reply:
that's the portal standard..

Reply:
see here for example https://github.com/liferay/liferay-portal/blob/7be65ed08473a8ce89548ce1eccc46e28eb3602a/modules/apps/headless/headless-admin-address/headless-admin-address-impl/rest-openapi.yaml#L243